test/built-ins/String/prototype/codePointAt/position-to-integer-less-than-zero-returns-undefined.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.3.3
+description: >
+  If pos < 0, return undefined
```

Comment:
This file (along with `position-to-integer-value-equal-or-more-than-size-returns-undefined.js`) tests two different steps (4 and 7) at once. It's probably better to keep the tests directed: one test for out-of-bounds numbers, one test for in-bounds numbers, and a final for string-to-number conversion. Admittedly, this will mean we won't have an explicit test for a negative number specified as a string, but that seems like a class of edge case that I don't think Test262 can hope to protect against.